tests/ResourceSyncingTest.php

```diff
@@ -126,6 +126,34 @@
     ], ResourceUser::first()->getAttributes());
 });
 
+test('creating the resource in tenant database creates it in central database and used custom attributes', function () {
+    // Assert no user in central DB
+    expect(ResourceUser::all())->toHaveCount(0);
+
+    $tenant = ResourceTenant::create();
+    pest()->artisan('tenants:migrate', [
+        '--path' => __DIR__ . '/Etc/synced_resource_migrations/custom',
+        '--realpath' => true,
+    ])->assertExitCode(0);
+
+    tenancy()->initialize($tenant);
+
+    // Create the same user in tenant DB
```

Comment:
What is "the same" user?

Reply:
I don't know, most probably leftover comments when writing tests. Actually, it should be only `// Create the user in tenant DB`.